<launch>
    <group>
        <remap from="/sensor/Imu" to="/part1/Imu"/>
        <node pkg="imu" type="imu_publisher_node.py" args="/dev/ttyUSB0" name="imu_part1"></node>
    </group>
    <group>
        <remap from="/sensor/Imu" to="/part2/Imu"/>
        <node pkg="imu" type="imu_publisher_node.py" args="/dev/ttyUSB1" name="imu_part2"></node>
    </group>
    <group>
        <remap from="/sensor/Imu" to="/part1/Imu"/>
        <remap from="/orientation" to="/part1/EKF"/>
        <node pkg="kalman_filter" type="kalman_filter.py" name="ekf_part1"></node>
    </group>
    <group>
        <remap from="/sensor/Imu" to="/part2/Imu"/>
        <remap from="/orientation" to="/part2/EKF"/>
        <node pkg="kalman_filter" type="kalman_filter.py" name="ekf_part2"></node>
    </group>
    <node pkg="kalman_filter" type="angle_error.py" name="angle_error"></node>
</launch>